configs/config.json.example

```diff
@@ -4,6 +4,7 @@
     "password": "YOUR_PASSWORD",
     "location": "SOME_LOCATION",
     "gmapkey": "GOOGLE_MAPS_API_KEY",
+    "libencrypt_location": "",
```

Comment:
Nope its set as encrypt="encrypt.so" or similar so having it not be set isnt a problem. If its "" it will check in the bots folder like usual